src/main/java/com/laserfiche/repository/api/clients/impl/model/FolderImportBody.java

```diff
@@ -0,0 +1,91 @@
+package com.laserfiche.repository.api.clients.impl.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.io.File;
+import java.util.Objects;
+
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FolderImportBody {
```

Comment:
I checked that this class isn't used anywhere. Does it need to be included?

Reply:
Thank you for catching the issue. I removed the class.